tests/kas_with_instance.spec.ts

```diff
@@ -163,6 +163,7 @@ test('test instance quick options', async ({ page }) => {
 
 // test_4kas.py test_kafka_dashboard_opened
 test('test instance dashboard on instance name click', async ({ page }) => {
+  await waitForKafkaReady(page, testInstanceName);
```

Comment:
I think that the relevant `import` is missing

Reply:
it's true! thanks

Reply:
@andreaTP added